### ISSUE TITLE
[React Native] Cannot read property 'findIndex' of undefined

### ISSUE THREAD

**Issue by xvonabur** [Gregivy/simpleddp#7]
Hello!

I'm trying to migrate my react native app from `react-native-meteor` to this library. Calling meteor methods via `call` works fine. Unfortunately I cannot subscribe to collection changes.

Every time collection changes I'm receiving an error:

![Simulator Screen Shot - iPhone X - 2019-04-25 at 15 08 18](https://user-images.githubusercontent.com/616857/56734620-07699680-676c-11e9-98df-affaf0c9ae20.png)

**Subscription**: 

```js
    let opts = {
       endpoint: props.url,
       SocketConstructor: WebSocket,
       reconnectInterval: props.reconnectInterval
    }
    this.meteorClient = new SimpleDDP(opts)
    let sub = this.meteorClient.sub('stream-room-messages', argsArr)

    sub.ready().then(() => {
      console.log(`Subscribed to ${eventName}`)
      sub.start()
      console.log(`${eventName} subscription is ON: ${sub.isOn()}`)
    })
```

**Getting collection**:

```js
    this.meteorClient.collection('stream-room-messages').reactive()

    subReactiveCursor.onChange(newData => {
      console.log(`new ${collectionName} data`, newData)
    })

   subReactiveCursor.data()

```

BTW `onChange` never called.

**Comment by Gregivy** [comment]
Hello, @xvonabur!

Are you sure that your collection has the same name as the subscription 'stream-room-messages'?

**Comment by Gregivy** [comment]
For example if your publication (lets name it 'mypub') returns a cursor like `return SomeCollection.find({})`, you should address `this.meteorClient.collection('somecollection')` with the same name you give to the collection when you declare a variable `const SomeCollection = new Mongo.Collection('somecollection')`. If your publication uses `this.added`, `

**Comment by xvonabur** [comment]
Yes, I'm sure about correct naming. I'm using `stream-room-messages` in subscription and collection with `react-native-meteor` and it works.

Basically, I'm using Rocket.Chat as a backend and react native app as livechat client. You can debug websocket messages in browser by going to https://chat-staging.ins-team.RU/livechat and using any name/email to start a chat.

P.S. I speak Russian, by the way :)

**Comment by Gregivy** [comment]
Thats nice to here :) But lets continue in english so others can understand if they need. Try to console log `this.meteorClient.collections` instead of using ` collection method. And I need to see your server side code of the publication.

**Comment by Gregivy** [comment]
I have released 2.0.1, let me know if it solves the problem.
